docs/guides/schemas.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Storage Schemas"
+linkTitle: "Storage Schemas"
+weight: 103
+slug: schemas
+---
+
+## Schema periodic table
```

Comment:
A couple of things:

1. I can't find any explanation of what a periodic table is and what the schema config is for within the Cortex doc. May be work adding an intro section explaining what's that. I've documented it into Loki some time ago (see this https://github.com/grafana/loki/blob/master/docs/operations/storage/table-manager.md) and I believe it could be a good starting point to write the Cortex one.
2. Not sure it's worth to mention CLI flags here. You can't configure any schema config option via CLI flag and we're about to remove them (https://github.com/cortexproject/cortex/issues/2208). Maybe explaining the config file is better.

Reply:
I've just moved existing docs here - I don't want updating them to block this PR. Therefore I've filled a docs issue for this: #2348